proto/wallirpc.proto

```diff
@@ -15,6 +16,13 @@ message GetInfoResponse {
   LndInfo lndInfo = 3;
 }
 
+message SubscribeToTxRequest {
+  bool reload = 1;
+  repeated string addresses = 2;
+  repeated string outpoints = 3;
+}
+message SubscribeToTxResponse {}
```

Comment:
I think your mistake is that `SubscribeToTxResponse` doesn't contain any field and therefore the client doesn't show anything when receiving the message. I could be wrong but it is still worth a try.